components/shared/ui/footer.tsx

```diff
@@ -10,8 +10,10 @@ export const Footer: FC<Props> = props => {
   const siteCodename = useSiteCodename();
 
   return (
-    <footer className={`w-full grow-0 h-16 px-10 flex flex-auto shrink-0 justify-end items-center ${mainColorBgClass[siteCodename]} ${props.className ?? ""}`}>
-      <p>Ficto Healthtech inc.</p>
+    <footer className={`${mainColorBgClass[siteCodename]} w-screen`}>
+      <div className="flex gap-4 items-center mx-auto max-w-screen-xl md:h-16">
```

Comment:
Why only set height on md? don't we want it to have some space in mobile devices?

![image](https://github.com/kontent-ai-bot/kontent-ai-new-sample-app/assets/54802833/95ec044c-a302-471a-ad14-0d4ef4e47fec)

Reply:
Will do

Reply:
9117424: set height for all breakpoints